deis-logger/manifests/deis-logger-svc.yaml

```diff
@@ -8,7 +8,8 @@ metadata:
     app: deis-logger
 spec:
   ports:
-  - port: 8088
+  - port: 80
+    targetPort: 8088
     name: http
     targetPort: http
```

Comment:
on second thought, I think it should be sufficient to leave this as-is and remove my `targetPort` addition. The result can then be

```yaml
- port: 80
  targetPort: http
```

thoughts?